### DISCUSSION TITLE
Turkish <-> English Machine Transtion

### DISCUSSION THREAD

**Discussion by YasmeenAbdelghany19** [NVIDIA-NeMo/NeMo#7300] [Q&A]
Hello fellow AI enthusiasts,

I hope you're all doing well. I'm currently working on a project involving English-Turkish machine translation and I'm seeking guidance on the best approach to take. As the NeMo toolkit provides a versatile platform for building and training NMT models, I'd appreciate your insights on the following matter:

 I'm exploring the feasibility of leveraging pre-trained models to kickstart the process. However, it seems that there are no readily available pre-trained models for this specific language pair.

Given this scenario, I'm wondering:

1. Is it possible to use any of the pre-trained models that could be fine-tuned for English-Turkish translation?
2. Alternatively, do I need to build the translation system from scratch, utilizing the model architectures described in the NeMo toolkit documentation?

I value your expertise and would greatly appreciate any insights, advice, or suggestions you could provide based on your experiences.

**Comment by titu1994** [comment]
@AlexGrinch @MaximumEntropy
